src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/CSIPReportRepository.kt

```diff
@@ -37,4 +37,6 @@ interface CSIPReportRepository : CrudRepository<CSIPReport, Long>, JpaSpecificat
     """,
   )
   fun findAllCSIPIds(pageable: Pageable): Page<Long>
+
+  fun findAllByOffenderBookingOffenderNomsId(offenderNo: String): List<CSIPReport>
```

Comment:
I could get the nomisId from the rootOffender on the CSIP Report and then thru to Offender but this is consistent with the other queries to get by offenderNo